packages/flutter/test/widgets/scrollable_test.dart

```diff
@@ -368,6 +368,82 @@ void main() {
     expect(getScrollOffset(tester), 20.0);
   });
 
+  group('setCanDrag to false with active drag gesture: ', () {
+    Future<void> pumpTestWidget(WidgetTester tester, bool canDrag) async {
+      return await tester.pumpWidget(
+        MaterialApp(
+          home: CustomScrollView(
+            physics: canDrag ? const AlwaysScrollableScrollPhysics() : const NeverScrollableScrollPhysics(),
+            slivers: <Widget>[SliverToBoxAdapter(
+                child: SizedBox(
+                  height: 2000,
+                  child: GestureDetector(onTap: () {},),
+                ),
+            )],
+          ),
+        ),
+      );
+    }
+
+    testWidgets('hold', (WidgetTester tester) async {
+      // Regression test for https://github.com/flutter/flutter/issues/66816.
+      await pumpTestWidget(tester, true);
+      final RenderIgnorePointer renderIgnorePointer = tester.renderObject<RenderIgnorePointer>(
+        find.descendant(of: find.byType(CustomScrollView), matching: find.byType(IgnorePointer)),
+      );
+
+      expect(renderIgnorePointer.ignoring, false);
+
+      final TestGesture gesture = await tester.startGesture(tester.getCenter(find.byType(Viewport)));
+      expect(renderIgnorePointer.ignoring, false);
```

Comment:
Shouldn't ignoring go to true when the gesture starts?

Reply:
Oh, I see. We haven't recognized the drag yet... maybe add a quick comment here why this is still false?

Reply:
Changed the name of the test to include the expected behavior.